README.md

```diff
@@ -4,808 +4,213 @@
-                    print("User audio format: \(userFormat)")
+                
+                Button("End Conversation", role: .destructive) {
+                    Task { await conversation.endConversation() }
+                }
+            } else {
+                Button("Start Voice Chat") {
+                    Task { await vm.startChat() }
                 }
+                .buttonStyle(.borderedProminent)
             }
-            .store(in: &cancellables)
-    }
-}
-```
-
-### Client Tool Support
-
-Handle tool calls from your agent with full parameter support:
-
-```swift
-private func handleToolCall(_ toolCall: ClientToolCallEvent) async {
-    do {
-        let parameters = try toolCall.getParameters()
-
-        let result = await executeClientTool(
-            name: toolCall.toolName,
-            parameters: parameters
-        )
-
-        if toolCall.expectsResponse {
-            try await conversation?.sendToolResult(
-                for: toolCall.toolCallId,
-                result: result
-            )
-        } else {
-            conversation?.markToolCallCompleted(toolCall.toolCallId)
         }
-    } catch {
-        // Handle tool execution errors
-        if toolCall.expectsResponse {
-            try? await conversation?.sendToolResult(
-                for: toolCall.toolCallId,
-                result: ["error": error.localizedDescription],
-                isError: true
-            )
-        }
-    }
-}
-
-private func executeClientTool(name: String, parameters: [String: Any]) async -> String {
-    switch name {
-    case "get_weather":
-        let location = parameters["location"] as? String ?? "Unknown"
-        return "Weather in \(location): 22°C, Sunny"
-
-    case "get_time":
-        return "Current time: \(Date().ISO8601Format())"
-
-    case "alert_tool":
-        return "User clicked something"
-
-    default:
-        return "Unknown tool: \(name)"
+        .padding()
     }
 }
 ```
 
-### Authentication Methods
+---
 
-#### Public Agents
+## Authentication Modes
 
-```swift
-let conversation = try await ElevenLabs.startConversation(
-    agentId: "your-public-agent-id",
-    config: ConversationConfig()
-)
-```
+### Public Agents
 
-#### Private Agents with Conversation Token
+Perfect for prototyping. Connect directly using your Agent ID from the ElevenLabs dashboard.
 
 ```swift
-// Option 1: Direct method (recommended)
-// Get a conversatoin token from your backend (never store API keys in your app)
-let token = try await fetchConversationToken()
-
-let conversation = try await ElevenLabs.startConversation(
-    conversationToken: token,
-    config: ConversationConfig()
-)
-
-// Option 2: Using auth configuration
-let conversation = try await ElevenLabs.startConversation(
-    auth: .conversationToken(token),
-    config: ConversationConfig()
-)
+let conversation = try await ElevenLabs.startConversation(agentId: "my-public-id")
```

Comment:
I know this is partly due to LiveKit internals but I believe we should separate creation and execution here as the conversation start task can be very slow and we are not giving the calling code a way to cancel without wrapping in a Task.

We can look at URLSession as a good example for how to design this API:

```
* create: let task = URLSession.shared.dataTask(with: url)
* start: task.resume()
* cancel: task.cancel()
```